helm/flowforge/values.yaml

```diff
@@ -45,7 +57,7 @@ forge:
     max: 1000
     timeWindow: 60000
 
-  domain: ""
+  domain: "example.com"
```

Comment:
We have left this empty in the past on purpose to force people to set it to something (so helm fails if it's not set)

This will mean the tests fail with the empty `default-values.ymal` and the `ci-values.ymal` will need to include `forge.domain`